src/routes/github/webhook/webhook-receiver-post.test.ts

```diff
@@ -70,39 +75,26 @@ describe("webhook-receiver-post", () => {
 	});
 
 	it("should throw an error if github app not found", async () => {
-		res = {
-			sendStatus: jest.fn()
-		};
 		req = createGHESReqForEvent("push", "", NON_EXIST_GHES_UUID);
 
-		await WebhookReceiverPost(req, res);
+		await WebhookReceiverPost(req, injectRawBodyToReq(res, req));
```

Comment:
Oh, missed this.  You need to move the `injectRawBodyToReq` to wrap the req only, and leave res alone :)

Reply:
Or maybe just remove that inject function. Modify the function the create the request object. Why bother a new function while the req was created for this testing purpose?